scripts/replace-aliases.py

```diff
@@ -3,15 +3,38 @@
 import json
 import glob
 import os
+import re
 import sys
 
 analysis_dir = sys.argv[1]
 alias_map_path = sys.argv[2]
 
 
-def replace_aliases(path, alias_map):
-    """Replace URL records with FILE records, based on the mapping extracted
-    from *.chrome-map.json files.
+def at_escape(text):
+  return re.sub("[^A-Za-z0-9_/]", lambda m: "@" + "{:02X}".format(ord(m.group(0))), text)
+
+
+def at_unescape(text):
+  return re.sub("@[0-9A-F][0-9A-F]", lambda m: chr(int(m.group(0)[1:], 16)), text)
+
+
+def resolve_relpath(url, relpath):
+    if url.startswith("chrome://"):
+        prefix = "chrome:/"
+        path = url[8:]
+    elif url.startswith("resource://"):
+        prefix = "chrome:/"
```

Comment:
If we're intentionally re-mapping resource:// to chrome:// here and this isn't a typo, then this probably merits a comment.

Reply:
Good catch!
it's just a wrong copy-paste.
I should've added test for resource URI as well